### ISSUE TITLE
Stats page

### ISSUE THREAD

**Issue by lmrodriguezr** [seq-code/registry#15]
We need a dedicated stats page to reduce confusion between name categories

**Comment by uqmchuvo** [comment]
Yes, and the confusion should be also resolved by addition of "?" after the word "registered" to the message under the "Search and explore Registered Names" (consider removing publications from there too?? to discuss). I found a message  "We have identified or registered" a bit confusing as all names that are auto-discovered are also registered, isn't it? Therefore, it should be just "We have registered? (that will explain what are the two categories under the registered names: auto-discovered or self-registered and validated through user registrations) XX names from XX publications by XX authors."
